test/module/irohad/consensus/yac/yac_mocks.hpp

```diff
@@ -32,6 +34,7 @@
 #include "consensus/yac/yac_gate.hpp"
 #include "consensus/yac/yac_hash_provider.hpp"
 #include "consensus/yac/yac_peer_orderer.hpp"
+#include "cryptography/crypto_provider/crypto_defaults.hpp"
```

Comment:
Cryptodefault is doesn't require for tests. Use mocks instead.

Reply:
I use it to generate keypair